### ISSUE TITLE
Does increasing max_decoder_steps affect on quality?

### ISSUE THREAD

**Issue by deepconsc** [NVIDIA/tacotron2#246]
I've question. 

We increased max_decoder_steps to 10^5, and as long as mels are generated from data during the training, and the data points are always smaller than max_decoder_steps, if it affect on the performance. 
P.S. We are fine-tuning on pre-trained model.

See the tensorboard logs:

Attention on 500 steps:
![individualImage](https://user-images.githubusercontent.com/48494620/61997892-3ec44800-b0b9-11e9-9bfe-0a4d859663f7.png)

Attention on 10500 steps (I'm not sure what the hell is happening here):
![individualImage-1](https://user-images.githubusercontent.com/48494620/61997897-500d5480-b0b9-11e9-8284-162af087b905.png)

Mel predicted on 10500 steps:
![individualImage-2](https://user-images.githubusercontent.com/48494620/61997905-5e5b7080-b0b9-11e9-983d-8a515fced360.png)

**Comment by rafaelvalle** [comment]
max_decoder_steps has no effect during training, only inference.
https://github.com/NVIDIA/tacotron2/blob/c76ac3b211fee3e958af316ce37ec4438c6d5626/model.py#L418

**Comment by deepconsc** [comment]
@rafaelvalle  Thank you!

And, could you explain the attention shown in 2nd image?

**Comment by rafaelvalle** [comment]
Hard to say without looking at your loss plots.

**Comment by rafaelvalle** [comment]
Closing due to inactivity.
